### PR TITLE
Umd

### PR DESCRIPTION
**Pull request by gkatsev**
Fixes #45

### CODE REVIEW

File: src/videojs.ads.js [hunk 1]
```diff
@@ -1,3 +1,13 @@
+(function(factory){
+    "use strict";
+    if (typeof define === 'function' && define.amd) {
+        define(['videojs'], function(vjs){factory(window, document, vjs);});
```
Comment (dmlap) [inline src/videojs.ads.js]: won't this register a named module called `videojs`?

Comment (gkatsev) [inline src/videojs.ads.js]: No, the array indicates that it's a dependency.

Comment (gkatsev) [inline src/videojs.ads.js]: Which reminds me, should we name this in the amd?
```js
define('videojs-contrib-ads', ['videojs'], function(vjs) { factory(window, document, vjs); });
```

**Comment by gkatsev** [comment]
The last break is from the prepublish script. Looking into correct action here.

File: src/videojs.ads.js (hunk 1 above)
Comment (dmlap) [inline src/videojs.ads.js]: Oops. Yeah, it makes sense as a dependency. I'm inclined to go with a named module so we don't run into the concatenation problem in downstream dependencies again.

**Comment by dmlap** [comment]
lgtm

**Pull request merged**